client/yarn.lock

```diff
@@ -3,28 +3,28 @@
 
 
 "@ag-grid-community/react@^25.2.0":
-  "integrity" "sha512-dNESmLUzAGDiKpSznKEVL+HoDULidgkFz24eukNOYRXo/TnsSVJSVfRdPe+be++oLEZLF3WnTjiAuExvIG2fvA=="
-  "resolved" "https://registry.npmjs.org/@ag-grid-community/react/-/react-25.2.0.tgz"
-  "version" "25.2.0"
+  version "25.2.0"
```

Comment:
Probably want to delete either this or package-lock and standardize on one of them